packages/ckeditor5-engine/src/view/downcastwriter.js

```diff
@@ -272,6 +273,36 @@ export default class DowncastWriter {
 		return uiElement;
 	}
 
+	/**
+	 * Creates a new {@link module:engine/view/rawelement~RawElement}.
+	 *
+	 *		writer.createRawElement( 'span' );
+	 *		writer.createRawElement( 'span', { id: 'foo-1234' } );
+	 *
+	 * Custom render function can be provided as third parameter:
+	 *
+	 *		writer.createRawElement( 'span', null, function( domDocument ) {
+	 *			const domElement = this.toDomElement( domDocument );
```

Comment:
IMO, in the callback you should get the DOM element as in case of RawElement, unlike like in UIElement, you do not control the host element. It's being controlled by the view.